src/main/kotlin/no/nav/klage/oppgave/api/OppgaveController.kt

```diff
@@ -29,4 +35,34 @@ class OppgaveController(val oppgaveService: OppgaveService) {
         return oppgaveService.getOppgaver()
     }
 
+    @GetMapping("/oppgaver/ikketildelt")
+    fun findIkkeTildelteOppgaver(): List<OppgaveView> {
+        logger.debug("findIkkeTildelteOppgaver is requested")
+        return oppgaveService.searchOppgaver(OppgaveSearchCriteria(erTildeltSaksbehandler = false))
+    }
+
+    @GetMapping("/oppgaver/tildelt/{ident}")
+    fun findTildelteOppgaver(@PathVariable(name = "ident", required = false) ident: String?): List<OppgaveView> {
+        logger.debug("findTildelteOppgaver is requested")
+        return oppgaveService.searchOppgaver(
+            OppgaveSearchCriteria(
+                erTildeltSaksbehandler = true,
+                saksbehandler = ident
+            )
+        )
+    }
+
+    @GetMapping("/oppgaver/tildelt/me")
+    fun findMineTildelteOppgaver(): ResponseEntity<List<OppgaveView>> {
```

Comment:
På tal om engelsk vs norsk, så vet ikke jeg om "mine tildelte" er domeneuttrykk, men jeg tror iaf ikke det? Tror vi burde ta en diskusjon i teamet om hvor vi skal legge oss i forhold til språk. Det viktigste for meg er at vi er enige om noe. Ellers blir det vanskelig å ta QA :)

Reply:
Fillern, jeg trodde jeg hadde fått det til jeg nå.. :) Min personlige mening er at det er rent kode-tekniske ting (som Controller og Service og sånt) som bør være engelsk, ting som MineTildelte kan gjerne være norsk. Men jeg prøver å tilpasse meg, og kan gjerne endre her også. Jeg prøvde å gå over og gjøre ting mer engelsk, men det glapp åpenbart her.

Reply:
Det er vel ikke definert nok rett og slett. Alternativt at vi sier det ikke skal defineres. At det blir opp til den som skriver :)